PycurlClient/src/python/RestClient/RestApi.py

```diff
@@ -1,7 +1,8 @@
 from RestClient.RequestHandling.HTTPRequest import HTTPRequest
-
+import pdb
```

Comment:
I guess this import is a leftover from your environment, right?

Reply:
correct! I'll fix it!